exercises/practice/linked-list/src/main/java/DoublyLinkedList.java

```diff
@@ -1,10 +1,29 @@
-/*
+class DoublyLinkedList<T> {
+    private Element<T> head;
 
-Since this exercise has a difficulty of > 4 it doesn't come
-with any starter implementation.
-This is so that you get to practice creating classes and methods
-which is an important part of programming in Java.
+    void push(T value) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
```

Comment:
```suggestion
        throw new UnsupportedOperationException("Please implement the DoublyLinkedList.push() method.");
```

Since exception messages appear in error messages, it is a good idea to make them give a good indication of what's actually missing.